TLM/TLM/UI/SubTools/VehicleRestrictionsTool.cs

```diff
@@ -248,15 +247,11 @@ public VehicleRestrictionsTool(TrafficManagerTool mainTool)
                 }
 
                 if (update) {
-                    updatedSegmentId = segmentId;
+                    break;
                 }
```

Comment:
So what's the point of update variable now?

Reply:
to break and avoid exception. maybe i should rename it to updated

Reply:
I'm asking because you've removed `RefreshCurrentRestrictedSegmentIds` located few lines below. Maybe I don't understand what is going on here, but I don't know how this little change (break) is preventing exception. Not to mention that `RefreshCurrentRestrictedSegmentIds` in this method(`ShowSigns`) was always(if `updatedSegmentId` changed) called **after** iteration

Reply:
There is no point to delay it anymore. It moved it to earlier point. See the diff at the bottom of the file.